packages/whats-new/src/whats-new-page.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { useEffect } from 'react';
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { GuidePage } from '@wordpress/components';
+import { useI18n } from '@automattic/react-i18n';
+
+function WhatsNewPage( {
+	pageNumber,
+	isLastPage,
+	alignBottom = false,
+	heading,
+	description,
+	imageSrc,
+	link,
+} ) {
+	const __ = useI18n().__;
+
+	useEffect( () => {
+		recordTracksEvent( 'calypso_block_editor_whats_new_slide_view', {
```

Comment:
FWIW event names are expected to be of the form: `<source>_<context>_<optional subcontext>_<action>_<optional qualifier>` - source: PCYsg-4S2-p2